packages/block-editor/src/components/rich-text/index.js

```diff
@@ -793,6 +820,9 @@ export class RichText extends Component {
 			}
 		}
 
+		// Wait for boundary class to be added.
+		setTimeout( () => this.recalculateBoundaryStyle() );
```

Comment:
This is not a safe timeout, despite the component being wrapped with `withSafeTimeout`. I suspect it may be the reason behind some failings I'm seeing in master E2E tests:

https://travis-ci.com/WordPress/gutenberg/jobs/187954533

```
FAIL packages/e2e-tests/specs/blocks/quote.test.js (41.221s)
  ● Quote › can be merged into from a paragraph
    TypeError: Cannot read property 'querySelector' of undefined
      at t.value (../../http:/localhost:8889/wp-content/plugins/gutenberg/build/block-editor/index.js?ver=1553629058:55:141215)
      at ../../http:/localhost:8889/wp-content/plugins/gutenberg/build/block-editor/index.js?ver=1553629058:55:144735
      at Suite.it (specs/blocks/quote.test.js:180:2)
```

Where one of the few occurrences of `querySelector` in `block-editor` is within `recalculateBoundaryStyle` ([referencing the component's `ref`](https://github.com/WordPress/gutenberg/blob/59b03c0c549619d8561e2d1c0d6d6c6b4c008d31/packages/block-editor/src/components/rich-text/index.js#L504), which would be [unset after unmount](https://github.com/WordPress/gutenberg/blob/59b03c0c549619d8561e2d1c0d6d6c6b4c008d31/packages/block-editor/src/components/rich-text/index.js#L169)). It seems reasonable that it could cause an error, if the component were to become unmounted after the `setTimeout` is scheduled.